### ISSUE TITLE
Where is the bam file for scATAC-seq data?

### ISSUE THREAD

**Issue by eravza** [GreenleafLab/MPAL-Single-Cell-2019#14]
Can you tell me How can I download the bam file for scATAC-seq data?
Thanks!

**Comment by jgranja24** [comment]
https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE139369

best

jeff

**Comment by eravza** [comment]
I already checked this website. Sorry for my ignorance but there are only fragments.tsv.gz files available for the scATAC on the website.

**Comment by jgranja24** [comment]
You are correct sorry im sending an email to GEO people as they didnt upload the bams correctly as we did here https://github.com/GreenleafLab/10x-scATAC-2019

**Comment by eravza** [comment]
Thank you! I saw the other study you pointed out. It is quite useful to find bam files in GEO.

**Comment by nikostrasan** [comment]
Hi here. May I ask if there is any update on this?
Unfortunately, I cannot find the bam files anywhere.
Also, the barcodes information is missing from the deposited fastq files, so it is impossible to run  **cellranger-atac count** to reproduce them from there.
Any suggestions ?
Thank you in advance

PS. From here:  https://kb.10xgenomics.com/hc/en-us/articles/360041540451--error-Entry-0-in-sample-defs-are-missing-input-FASTQs ,  seems that R2 | 10x barcode (i5) fastq file is missing from all deposited SRA samples.

**Comment by lingfeiwang** [comment]
Could you update the issue when the raw bam files become available? Thanks, @jgranja24 .

**Comment by Changtianpeng** [comment]
Could you update the issue when the raw bam files become available? Thanks.@ @jgranja24

**Comment by Changtianpeng** [comment]
Hi. have you get the bam files? @lingfeiwang @nikostrasan @eravza
